src/future-hopr-lib-components/Layout/drawer.tsx

```diff
@@ -1,156 +1,147 @@
-import * as React from 'react';
-import Box from '@mui/material/Box';
-import Divider from '@mui/material/Divider';
-import Drawer from '@mui/material/Drawer';
-import { Link, LinkProps } from 'react-router-dom';
-import { useLocation } from 'react-router-dom';
 import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+import {
+  Divider,
+  List,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  ListSubheader,
+  Tooltip
+} from '@mui/material'
+import MuiDrawer from '@mui/material/Drawer';
+import { useEffect, useState } from 'react';
+import { Link, useLocation } from 'react-router-dom';
+import { ApplicationMapType } from '../../router';
 
-import List from '@mui/material/List';
-import ListItem from '@mui/material/ListItem';
-import ListItemButton from '@mui/material/ListItemButton';
-import ListItemIcon from '@mui/material/ListItemIcon';
-import ListItemText from '@mui/material/ListItemText';
-import Toolbar from '@mui/material/Toolbar';
-import Typography from '@mui/material/Typography';
+const drawerWidth = 240;
+const minDrawerWidth = 56;
 
-// Types
-import type { ApplicationMapType } from '../../router';
+const StyledDrawer = styled(MuiDrawer)`
+  .MuiDrawer-paper {
+    box-sizing: border-box;
+    padding-top: 59px;
+    transition: width 0.4s ease-out;
+    overflow: hidden;
+    width: ${(props) => (props.open ? `${drawerWidth}px` : `${minDrawerWidth}px`)};
 
-const drawerWidth = 240;
+    ${(props) =>
+    props.variant === 'temporary' &&
+      css`
+        width: ${drawerWidth}px;
+      `}
+  }
+`;
 
-interface Props {
-  /**
-   * Injected by the documentation to work in an iframe.
-   * You won't need it on your project.
-   */
-  window?: () => Window;
-  drawerItems?: any[];
+const StyledListSubheader = styled(ListSubheader)`
+  align-items: center;
+  color: #777;
+  display: flex;
+  height: 64px;
+  letter-spacing: 0.2px;
+  user-select: none;
+`;
+
+type DrawerProps = {
+  drawerItems: ApplicationMapType;
   drawerLoginState?: {
-    node?: boolean;
-    web3?: boolean;
+    node: boolean;
+    web3: boolean;
```

Comment:
With this change, we will not be able to build the app only as a Node Admin or Staking UI. Please put it back